ipyvolume/embed.py

```diff
@@ -140,63 +57,118 @@ def save_js(dirname, makedirs=True):
     src = os.path.join(dir_name_src, "index.js")
     shutil.copy(src, dst)
```

Comment:
You'r right, it's not index.js in the static dir. I think we can do this smart. If we are in dev mode (ipywidgest module is a symlink), the like should be at <ipywidgets.__file__>../js/dist/index.js if that directory doesn't exists, we are not in dev mode and we should fetch it online. The other option would be to include the file (say ipyvolume.js in the same static folder as well).